docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,27 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| WorkflowId                 | Keyword  | Identifies the Workflow Execution.                                                                                                                             |
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                 |
+| WorkflowType               | Keyword  | The type of Workflow.                                                                                                                                          |
+| ExecutionStatus            | Keyword  | The current state of the Workflow Execution.                                                                                                                   |
+| StartTime                  | Datetime | The time at which the Workflow Execution started.                                                                                                              |
+| CloseTime                  | Datetime | The time at which the Workflow Execution completed.                                                                                                            |
+| ExecutionTime              | Datetime | Same as StartTime for the most cases but different for Cron Workflows and retried Workflows; the time at which the Workflow Execution actually begins running. |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution (in nanoseconds). Available only for closed Workflows.                                                           |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                      |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                |
+| HistorySizeBytes           | Long     | The size of the Event History.                                                                                                                                 |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                         |
+| TemporalChangeVersion      | Keyword  | Stores change/version pairs if the GetVersion API is enabled.                                                                                                  |
+| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                 |
```

Comment:
```suggestion
| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution. Deprecated since server version 1.21 in favor of the `BuildIds` search attribute.                                                                                               |
```